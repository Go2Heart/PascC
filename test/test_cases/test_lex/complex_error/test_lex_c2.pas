program test(input,output);  
const a=''  
      b='abc  
