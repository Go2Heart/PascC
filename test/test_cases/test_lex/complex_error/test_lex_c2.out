LexToken(PROGRAM,'program',1,0)
LexToken(ID,'test',1,8)
LexToken(LPAREN,'(',1,12)
LexToken(ID,'input',1,13)
LexToken(COMMA,',',1,18)
LexToken(ID,'output',1,19)
LexToken(RPAREN,')',1,25)
LexToken(SEMI,';',1,26)
LexToken(CONST,'const',2,30)
LexToken(ID,'a',2,36)
LexToken(EQU,'=',2,37)
LexToken(CCONST,"''",2,38)
LexToken(ID,'b',3,49)
LexToken(EQU,'=',3,50)
LexToken(ID,'abc',3,52)
